kit/popups/src/Sheets/UIFullscreenSheet.tsx

```diff
@@ -11,13 +12,17 @@ export type UIFullscreenSheetProps = UISheetProps & {
 
 export function UIFullscreenSheet({ children, style, ...rest }: UIFullscreenSheetProps) {
     const {
-        screen: { height },
+        screen: { height: screenHeight },
+        window: { height: windowHeight },
     } = useDimensions();
 
-    const fullscreenHeight = React.useMemo(
-        () => height + UILayoutConstant.rubberBandEffectDistance,
-        [height],
-    );
+    const fullscreenHeight = React.useMemo(() => {
+        if (Platform.OS === 'web') {
+            return windowHeight + UILayoutConstant.rubberBandEffectDistance;
+        }
```

Comment:
Why should it be applied only for web?
Why not using `windowHeight` for all platforms here?